### DISCUSSION TITLE
关于从头开始训练的问题

### DISCUSSION THREAD

**Discussion by linforgithub** [Equim-chan/Mortal#67] [Q&A]
我先执行了train.py,有数据之后，我开始执行train.py,然后遇到了需要加载TestPlayer的问题
![image](https://github.com/Equim-chan/Mortal/assets/16068620/31f1ae53-79de-434f-95c9-3d9ed4970631)
于是我把从文件加载的注释掉了，但是这个文件会在什么时候生成呢，执行了train好久，也没有生成Player需要的文件，也就是说我如果需要执行对局之类的操作是没法进行的

**Comment by Equim-chan** [comment]
这个文件就是字面上的 state file，和你训练的产物是同一种类型的文件。

实际上从头开始训练 Mortal 需要做 bootstrap，即要训练 Mortal 模型首先还需要另一个 Mortal 模型作为 test play 的对手，只是现在的代码里没有包含这一段，要手动修改操作。你可以在训练一段时间之后把 state file 归档一份，然后将 test play 的 state file 设置成它。

**Reply by linforgithub** [reply]
也就是说这个文件就是我下图配置的state_file，也就是train.py执行后的产物是吧。
![image](https://github.com/Equim-chan/Mortal/assets/16068620/5b03a6ff-06f6-476a-aa63-8f5e29d68536)

然后还有一个问题：
INFO one_vs_three.rs:131  seed: [10000, 10750) w/ 0x2000, start 750 groups, 3000 hanchans
我跑完train.py都会有一个上边的输出，这个是正常的是么，还是我哪些设置不对呢
